models/DataObject/Listing.php

```diff
@@ -178,7 +178,7 @@ public function count()
      * @param int $offset
      * @param int $itemCountPerPage
      *
-     * @return array
+     * @return int
```

Comment:
I think `@return array` was correct

Reply:
Yep, count should return `int`

Reply:
In that `case @method int load()` should be `@method array load()`.

I assumed the higher order one was the correct one.

Reply:
(line 26 by the way)

Reply:
In Line 26 is:
https://github.com/pimcore/pimcore/blob/50c71ce7ff1cc75caff5b427b8a09cdb373b434b/models/DataObject/Listing.php#L26

You can change it here also to `@return Model\DataObject[]`